test/smoke/testApps/AgentExceptionAndDependency/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="SimpleCalculator_1" version="3.0">
+  <display-name>SimpleCalculator</display-name>
+  <filter>
+    <filter-name>ApplicationInsightsWebFilter</filter-name>
+    <filter-class>com.microsoft.applicationinsights.web.internal.WebRequestTrackingFilter</filter-class>
+  </filter>
```

Comment:
Do we need this anymore. I think filter already has the annotation which takes care of this.

Reply:
I will check this.